Frontend/ui-customer/src/Components/EmergencyHelp/View.purs

```diff
@@ -411,39 +411,36 @@ allContactsView state push =
                 , textSize FontSize.a_12
                 ]
               ]
-              ,  linearLayout
-                 [ height  WRAP_CONTENT
-                 , width  WRAP_CONTENT
-                 , padding (PaddingLeft 8)
-                 ][  textView
-                   [text (item.name)
-                   , color Color.black800
-                   , textSize FontSize.a_16
-                   , lineHeight "20"
-                   , fontStyle $ FontStyle.semiBold LanguageStyle
-                   ]
-                  ]
-              ,
-              linearLayout
-              [ height WRAP_CONTENT
-              , width MATCH_PARENT
-              , gravity RIGHT
-              ][
-                linearLayout
-                  [ height  WRAP_CONTENT
-                  , width  WRAP_CONTENT
-                  , onClick push $ const $ CallContactPopUp item
-                  ][ textView
+          , linearLayout
+            [ height WRAP_CONTENT
+            , weight 1.0
+            , padding (PaddingLeft 8)
+            ][  textView
+                [ text (item.name)
+                , color Color.black800
+                , textSize FontSize.a_16
+                , lineHeight "20"
+                , fontStyle $ FontStyle.semiBold LanguageStyle
+                ]
+            ]
+          , linearLayout
+            [ height WRAP_CONTENT
+            , width $ V 50
+            , gravity CENTER
+            ][  linearLayout
+                [ height  WRAP_CONTENT
+                , width  WRAP_CONTENT
+                , onClick push $ const $ CallContactPopUp item
+                ][ textView
                     [ text $ (getString CALL)
                     , color Color.green900
                     , textSize FontSize.a_14
                     , lineHeight "18"
                     , fontStyle $ FontStyle.regular LanguageStyle
-                    , margin $ MarginLeft 5
-                    , padding $ Padding 20 10 20 10
+                    , margin $ Margin 10 0 10 0
```

Comment:
@Anuragini-2506 can we use MarginHorizontal instead

Reply:
Done